src/plugins/OauthPlugin/Oauth/OauthPkceStrategy.js

```diff
@@ -22,45 +19,24 @@ export default class OauthPkceStrategy {
     this.codeVerifier = '';
   }
 
-  // TODO: after implement public api use this method and drop dialog
-  // /**
-  //  *
-  //  * @private
-  //  * @param {string} oauthServer
-  //  * @param {object} fields
-  //  * @return {Promise<object>}
-  //  */
-  // static async exchangeCodeToToken(oauthServer, fields) {
-  //   const formData = Object.keys(fields).reduce((form, key) => {
-  //     form.append(key, fields[key]);
-  //     return form;
-  //   }, new FormData());
-  //
-  //   const url = `${oauthServer}/token`;
-  //
-  //   const { data } = await axios.post(url, formData);
-  //   return data;
-  // }
-
   /**
    *
    * @private
+   * @param {string} oauthServer
    * @param {object} fields
    * @return {Promise<object>}
    */
-  async exchangeCodeToToken(fields) {
-    const { payload, status, error } = await this.context.ask(
-      MESSENGER_METHODS.EXCHANGE_TOKEN_REQUEST,
-      fields,
-    );
+  async exchangeCodeToToken(oauthServer, fields) {
+    const formData = Object.keys(fields).reduce((form, key) => {
```

Comment:
I think `acc` is much cleaner, because in `reduce` we are always working with accumulators

Reply:
acc in more abstract, than form